src/convention.js

```diff
@@ -1,16 +1,14 @@
 module.exports = class Convention {
-
-    static camelToSnake(string) {
-        return string.replace(/([A-Z])/g, "_$1").toLowerCase()
-    }
-
-    static toTableFieldName(entityField) {
-        return this.camelToSnake(entityField)
-    }
-
-    static isScalarType(type) {
-        const scalarTypes = [Number, String, Boolean, Date, Object, Array]
-        return scalarTypes.includes(type)
-    }
-
-}
\ No newline at end of file
+  camelToSnake (string) {
+    return string.replace(/([A-Z])/g, '_$1').toLowerCase()
+  }
+
+  toTableFieldName (entityField) {
+    return this.camelToSnake(entityField)
+  }
+
+  isScalarType (type) {
```

Comment:
does make sense this checker be inside suma?

Reply:
@jhomarolo this function was already used in the DataMapper Proxy.

https://github.com/maikmb/herbs2knex/blob/master/src/dataMapper.js#L93

Reply:
I'll aprove this, but please open a issue in suma to transfer this checker there.